clio-client/src/main/scala/org/broadinstitute/clio/client/util/IoUtil.scala

```diff
@@ -38,20 +38,62 @@ object IoUtil {
       .map[File](path => path.toFile)
       .forEach(file => file.deleteOnExit())
   }
+
+  def moveGoogleObject(from: String, to: String): Int = {
+    val gs = new GsUtil(None)
+    gs.mv(from, to)
```

Comment:
Didn't we talk about using the java bindings here instead of shelling out to call gsutil, to avoid the various issues we've run into with gsutil elsewhere?